src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -91,15 +91,22 @@ public void NavigateToPage(Uri link)
             this.documentationBrowser.NavigateToString(this.viewModel.GetContent());
         }
 
-        /// <summary>
-        /// Dispose function for DocumentationBrowser
-        /// </summary>
-        public void Dispose()
+        protected virtual void Dispose(bool disposing)
         {
+            // Cleanup
             this.viewModel.LinkChanged -= NavigateToPage;
             this.documentationBrowser.Navigating -= ShouldAllowNavigation;
             this.documentationBrowser.Dispose();
             this.documentationBrowser.DpiChanged -= DocumentationBrowser_DpiChanged;
         }
+
+        /// <summary>
+        /// Dispose function for DocumentationBrowser
+        /// </summary>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
@mjkkirschner Reminded me why we are not doing this. This make me wonder if it's worth to implement the Dispose pattern at all or just leave them as it is and look for a better solution.